backend/src/v1/services/external-consumer-service.ts

```diff
@@ -11,14 +11,22 @@ import { Prisma } from '@prisma/client';
 
 const DEFAULT_PAGE_SIZE = 50;
 
+const withStartOfDay = (input: LocalDateTime): LocalDateTime => {
+  return input.withHour(0).withMinute(0).withSecond(0).withNano(0);
+};
+
+const withEndOfDay = (input: LocalDateTime): LocalDateTime => {
+  return input.withHour(23).withMinute(59).withSecond(59);
+};
+
 const externalConsumerService = {
   /**
    * This function returns a list of objects with pagination details to support the analytics team.
-   * this endpoint should not return more than 1000 records at a time.
-   * if limit is greater than 1000, it will default to 1000.
-   * calling this endpoint with no limit will default to 1000.
+   * this endpoint should not return more than 50 records at a time.
+   * if limit is greater than 50, it will default to 50.
+   * calling this endpoint with no limit will default to 50.
    * calling this endpoint with no offset will default to 0.
-   * calling this endpoint with no start date will default to 30 days ago.
+   * calling this endpoint with no start date will default to yesterday.
```

Comment:
are we not supposed to work at -1 days?

Reply:
add to integration test scenarios

Reply:
Yes, I was manually testing this last night, and I noticed diff results between the unit tests and swagger results